gnowledge/gstudio
# Making first-time-log-in user and existing users member of home group

**Making first-time-log-in user and existing users member of home group:**
- To make member of home group for first time log-in user:
  - As on when user's `Author` object gets created, to very next line in `home.py` user's `id` gets added to `home` group's `author_set`.
- To add existing user's `id` into `home` group, code added in `sync_existing_docments.py` file.

--

**TEST:**
- Check whether existing user get's added to home group or not:
  - After taking the pull run script `sync_existing_documents.py`:
  - `python manage.py sync_existing_documents`
  - Check for change in the member's count on LHS of `home` group's detail view.
- Check whether newly registered user is part of `home` group or not:
  - Do fresh registration of new user. Activate and log-in.
  - To check user's membership, click on `List Member's` option on RHS. Wherein listing will appear, which should contain username of newly log-in member.
